deploy/cloudformation/manifest-passthrough-pipeline.yml

```diff
@@ -0,0 +1,701 @@
+    Type: String
+    Default: "/all/marble-passthroughprimo-pipeline"
+    Description: The path the keys for parameter store should be read and written to for config
+  AppConfigPathTest:
+    Type: String
+    Default: "/all/marble-passthroughprimo-pipeline-test"
+    Description: The path the keys for parameter store should be read and written to for config
+  PassthroughUrl:
+      NoEcho: true
+      Type: String
+      Description: Primo passthrough url
+  PrimoApiKey:
+      NoEcho: true
+      Type: String
+      Description: Primo API key
+  PrimoSandboxApiKey:
+      NoEcho: true
+      Type: String
+      Description: Sandbox Primo API key
+  PassthroughVersion:
+    NoEcho: true
+    Type: String
+    Description: Passthrough Git versioning information
+
+Resources:
+  CodeBuildTrustRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - codebuild.amazonaws.com
+            Action: 'sts:AssumeRole'
+      Path: /
+  CodeBuildRolePolicy:
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyName: CodeBuildRolePolicy
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - 'logs:CreateLogGroup'
+              - 'logs:CreateLogStream'
+              - 'logs:PutLogEvents'
+            Resource: !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/${AWS::StackName}-*'
+          - Effect: Allow
+            Resource: !Sub
+              - '${PipelineBucketArn}/*'
+              - PipelineBucketArn: !GetAtt S3Bucket.Arn
+            Action:
+              - 's3:GetObject'
+              - 's3:PutObject'
+      Roles:
+        - !Ref CodeBuildTrustRole
+  CloudFormationTrustRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - cloudformation.amazonaws.com
+            Action: 'sts:AssumeRole'
+      Path: /
+  CloudFormationRolePolicy:
+    Type: 'AWS::IAM::Policy'
+    Properties:
+      PolicyName: CloudFormationRolePolicy
+      PolicyDocument:
+        Statement:
+          # Allow the role to create SSM resources specified by manifest-pipeline.yml
+          - Action:
+            - 'ssm:Delete*'
+            - 'ssm:Put*'
+            - 'ssm:AddTags*'
+            Resource:
+              - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/all/stacks/${TestStackName}/*"
+              - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/all/stacks/${ProdStackName}/*"
+              - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${AppConfigPathProd}/*"
+              - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${AppConfigPathTest}/*"
+            Effect: Allow
+          - Action:
+              - 'lambda:*'
```

Comment:
We're trying to use less wildcards on actions, so change this to the explicit list.